xml/admin_ceph_gateway.xml

```diff
@@ -2692,349 +2692,279 @@ rgw_override_bucket_index_max_shards = 12
-  </sect2>
-
-  <sect2 xml:id="ceph-rgw-fed-naming">
-   <title>Naming Conventions</title>
-   <para>
-    This example describes the process of setting up a master zone. We will
-    assume a zonegroup called <literal>us</literal> spanning the United States,
-    which will be our master zonegroup. This will contain two zones written in
-    a <replaceable>ZONEGROUP</replaceable>-<replaceable>ZONE</replaceable>
-    format. This is our convention only and you can choose a format that you
-    prefer. In summary:
-   </para>
-   <itemizedlist>
-    <listitem>
-     <para>
-      Master zonegroup: United States <literal>us</literal>
-     </para>
-    </listitem>
-    <listitem>
-     <para>
-      Master zone: United States, East Region 1: <literal>us-east-1</literal>
-     </para>
-    </listitem>
-    <listitem>
-     <para>
-      Secondary zone: United States, East Region 2:
-      <literal>us-east-2</literal>
-     </para>
-    </listitem>
-    <listitem>
-     <para>
-      Secondary zone: United States, West Region: <literal>us-west</literal>
-     </para>
-    </listitem>
-   </itemizedlist>
-   <para>
-    This will be a part of a larger realm named <literal>gold</literal>. The
-    <literal>us-east-1</literal> and <literal>us-east-2</literal> zones are
-    part of the same &ceph; cluster, <literal>us-east-1</literal> being the
-    primary one. <literal>us-west</literal> is in a different &ceph; cluster.
-   </para>
-  </sect2>
-
-  <sect2 xml:id="ceph-rgw-fed-pools">
-   <title>Default Pools</title>
-   <para>
-    When configured with the appropriate permissions, &ogw; creates default
-    pools on its own. The <literal>pg_num</literal> and
-    <literal>pgp_num</literal> values are taken from the
-    <filename>ceph.conf</filename> configuration file. Pools related to a zone
-    by default follow the convention of
-    <replaceable>ZONE-NAME</replaceable>.<replaceable>POOL-NAME</replaceable>.
-    For example, for the <literal>us-east-1</literal> zone, it will be the
-    following pools:
-   </para>
-<screen>.rgw.root
-us-east-1.rgw.control
-us-east-1.rgw.data.root
-us-east-1.rgw.gc
-us-east-1.rgw.log
-us-east-1.rgw.intent-log
-us-east-1.rgw.usage
-us-east-1.rgw.users.keys
-us-east-1.rgw.users.email
-us-east-1.rgw.users.swift
-us-east-1.rgw.users.uid
-us-east-1.rgw.buckets.index
-us-east-1.rgw.buckets.data
-us-east-1.rgw.meta</screen>
-   <para>
-    These pools can be created in other zones as well, by replacing
-    <literal>us-east-1</literal> with the appropriate zone name.
-   </para>
-  </sect2>
-
-  <sect2 xml:id="ceph-rgw-fed-realm">
+  <sect3 xml:id="ceph-rgw-fed-realm">
    <title>Creating a Realm</title>
+   <para>A realm represents a globally unique namespace consisting of one or
+     more zonegroups containing one or more zones, and zones containing buckets,
+     which in turn contain objects. A realm enables the &ceph; &ogw; to
```

Comment:
The "and" in the middle of a lengthy sentence is awkward. I suggest breaking the sentence into 2 smaller ones at "and".